Framework/Algorithms/src/NormaliseToMonitor.cpp

```diff
@@ -657,19 +656,23 @@ void NormaliseToMonitor::normaliseBinByBin(
 /** Calculates the overall normalization factor.
  *  This multiplies result by (bin width * sum of monitor counts) / total frame
  * width.
- *  @param X The X vector
- *  @param Y The data vector
- *  @param E The error vector
+ *  @param X The BinEdges of the workspace
+ *  @param Y The Counts of the workspace
+ *  @param E The CountStandardDeviations of the workspace
  */
-void NormaliseToMonitor::normalisationFactor(const MantidVec &X, MantidVec *Y,
-                                             MantidVec *E) {
-  const double monitorSum = std::accumulate(Y->begin(), Y->end(), 0.0);
+void NormaliseToMonitor::normalisationFactor(const BinEdges &X, Counts &Y,
```

Comment:
This function seems to do nothing else but converting a histogram to `YMode::Frequencies` and a normalization. @LamarMoore, maybe such a normalization is something we want to add to `HistogramData`?